DllImportGenerator/DllImportGenerator/TypePositionInfo.cs

```diff
@@ -278,6 +310,28 @@ static bool TryCreateTypeBasedMarshallingInfo(ITypeSymbol type, Compilation comp
             }
         }
 
+        private static ByValueContentsMarshalKind GetByValueContentsMarshalKind(IEnumerable<AttributeData> attributes, Compilation compilation)
+        {
+            INamedTypeSymbol outAttributeType = compilation.GetTypeByMetadataName(TypeNames.System_Runtime_InteropServices_OutAttribute)!;
```

Comment:
Can we return early if either of these are `null`? I'm not arguing for it, just making sure I understand these could be null. If not, are they never `null` since they are defined in SPCL or the like?

Reply:
These types are defined in CoreLib, so they'll never be null.